src/Transport/Addressing/Sanitization/Strategies/ValidateAndHashIfNeeded.cs

```diff
@@ -125,6 +129,7 @@ internal ValidateAndHashIfNeeded(ReadOnlySettings settings)
             }
         }
 
+        /// <summary></summary>
```

Comment:
empty

Reply:
Find it redundant to explain the obvious, but will change.